server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/UpdateAuditInfo.java

```diff
@@ -41,6 +41,13 @@
  */
 @SuppressWarnings({"PMD.AtLeastOneConstructor", "PMD.CyclomaticComplexity"})
 public class UpdateAuditInfo extends AbstractDoSDashboardEndpoint {
+  /**
+   * Uploaded file attribute
+   *
+   * Factored out due to the PMD rule AvoidDuplicateLiterals
```

Comment:
We'll probably run into an easily resolvable conflict around `UPLOAD_FILE_AFTER`; #37 includes something similar.